clothing/dashboard_views.py

```diff
@@ -25,3 +31,54 @@ class OrderOverView(ClothingAppMixin, ListView):
 
     def get_queryset(self):
         return Order.get_current()
+
+
+class FreeClothingView(ClothingAppMixin, ListView):
+    model = Person
+    template_name = "clothing/dashboard/orders_person_free.html"
+    context_object_name = "persons"
+
+    def get_queryset(self):
+        return Person.get_current()
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data()
+        context['persons'] = context['persons'].annotate(
+            num_free_orders=models.Sum(models.Case(
+                models.When(order__additional=False, then=1),
+                default=0,
+                output_field=models.IntegerField()
+            ))
+        )
+        return context
+
+
+class OrderAggregatedView(ClothingAppMixin, TemplateView):
+    template_name = "clothing/dashboard/orders_aggregated.html"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data()
+
+        types = Type.objects.all()
+        colors = Color.objects.all()
+        sizes = Size.objects.all()
+
+        context['orders'] = OrderedDict()
+        for additional_type, additional_name in [(False, _('Kostenlos')), (True, _('Zusätzlich'))]:
+            # Create bins
+            context['orders'][additional_name] = OrderedDict()
+            for color in colors:
+                context['orders'][additional_name][color.name] = OrderedDict()
+                for type in types:
+                    context['orders'][additional_name][color.name][type.name] = OrderedDict()
+                    for size in sizes:
+                        context['orders'][additional_name][color.name][type.name][size.size] = 0
+
+            # Store orders
+            orders = Order.get_current().filter(additional=additional_type)
+            for order in orders:
+                context['orders'][additional_name][order.color.name][order.type.name][order.size.size] += 1
+
+        print(context['orders'])
```

Comment:
It creates and fills the histogram bins for an aggregated order overview like this:

![aggregated](https://cloud.githubusercontent.com/assets/6319964/17836257/fc49df0e-678b-11e6-9bdf-74783aa2cdda.png)